pkg/subctl/cmd/show.go

```diff
@@ -98,3 +104,20 @@ func getClientConfigAndClusterName(rules *clientcmd.ClientConfigLoadingRules, ov
 
 	return restConfig{config: clientConfig, clusterName: *clusterName}, nil
 }
+
+func getSubmarinerResource(config *rest.Config) *v1alpha1.Submariner {
+	submarinerClient, err := submarinerclientset.NewForConfig(config)
+	exitOnError("Unable to get the Submariner client", err)
+
+	submariner, err := submarinerClient.SubmarinerV1alpha1().Submariners(OperatorNamespace).Get(submarinercr.SubmarinerName, v1.GetOptions{})
+	if err != nil {
+		if errors.IsNotFound(err) {
+			fmt.Println("Submariner is not installed")
```

Comment:
Can you move this out of the function after the call to getSubmarinerResource ?
Thanks